Templates/ExternalFileTrigger-FSharp/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": ExternalFileTrigger",
+    "name": "ExternalFileTrigger",
     "description": "$SaasFileTriggerFSharp_description",
```

Comment:
Those name changes were made in a different PR - I'd rather not mix solution file maintenance with this one :)